components/ingest-service/server/chef.go

```diff
@@ -398,6 +398,11 @@ func (s *ChefIngestServer) StartReindex(ctx context.Context, req *ingest.StartRe
 			log.WithFields(log.Fields{"index": index}).WithError(err).Error("Failed to delete temporary index: ", tempIndex)
 			continue
 		}
+
+	}
+
+	if err := s.db.UpdateReindexRequest(requestID, STATUS_COMPLETED, time.Now()); err != nil {
```

Comment:
You have to define a variable, say isFailed, and the default value is false. Whenever there is a failure in any of the above stages, you have to update the variable as true, and here based on the value you need to update the status

Reply:
Yes. Made the changes.